.github/workflows/native-image.yml

```diff
@@ -0,0 +1,50 @@
+name: Native Image
+
+on:
+  push:
+    branches:
+      - native-image
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  workflow_config:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+
+      - name: Get jar
+        run: |
+          wget "https://github.com/plantuml/plantuml/releases/download/${PLANTUML_VERSION}/plantuml-${PLANTUML_VERSION#?}.jar" -O plantuml.jar
+        env:
+          PLANTUML_VERSION: "v1.2023.4"
+
+      - name: Install system requirements
+        run: sudo apt install build-essential libz-dev zlib1g-dev
+
+      - name: Get GraalVM
+        run: |
+          wget "https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-${GRAALVM_VERSION}/graalvm-ce-java17-linux-amd64-${GRAALVM_VERSION}.tar.gz" -O graalvm-ce.tar.gz
+          mkdir graalvm-ce
+          tar xvzf graalvm-ce.tar.gz -C graalvm-ce --strip-components 1
+          ./graalvm-ce/bin/gu install native-image
+        env:
+          GRAALVM_VERSION: "22.3.1"
```

Comment:
Another minor question:
- Should we take only version `22.3.1` of `GraalVM`
- or should we take the latest?

_For `plantuml.jar` that is OK, we will plan to use the current version, the `wip` of the current CI himself._


Regards.

Reply:
> Should we take only version 22.3.1 of GraalVM

I think we should use a fixed version and manually update it (with careful consideration).

> For plantuml.jar that is OK, we will plan to use the current version, the wip of the current CI himself.

👍🏻